### PR TITLE
Clarify use of Other Login Method

### LINKED ISSUE #548: Clarify use of alternate login methods in Multi-Factor Login
The [Multi-Factor Login](http://www.patternfly.org/pattern-library/application-framework/multi-factor-login/) documentation lacks clarity around use of "Other Login Methods." There are two primary issues that I see:

(1) It's not clear where these screens fit into the overall flow. After the system validates the username, do these screens come next (i.e. before the password), after the password screen, or in place of the password. It might be helpful to include a flow chart or similar diagram that shows this sequence.

(2) It's unclear why the Other Login Method screens have a user icon in front of the username but the password screen does not. What's the difference? Also, is this intended to just be a static icon or an avatar? If the later, I think it should be marked optional since all underlying systems may not support that. If we intend to keep this, should add a callout to describe what it is.

### PR DESCRIPTION
**Pull request by yih-wang**
Closes #548

### CODE REVIEW

File: pattern-library/application-framework/multi-factor-login/design/design.md [hunk 1]
```diff
@@ -13,17 +13,13 @@
...
-## Other Login Methods
-![Image of other login methods](img/login-page-5.png)
-1. **Verification Code**: User can bind the authenticator which installed on their mobile phones with their accounts in registration process. If so, they can login with the code that authenticators provide.
-2. **SMS Code**: Users can bind their phone number (or backup phone numbers) with their accounts in registration process. If so, they can login with SMS.
-3. **Backup Code**: A backup code (random strings) can be provided to users in registration process. If so, they can login with backup code.
-4. **More Verification Options**: Users can switch between different methods through this link if they can't finish the current one. The link will lead to followed screen.
-
-![Image of more verification options](img/more-verification-options.png)
+## Multi-Factor Login Example
+![Image of other login methods](img/Example.png)
```
Comment (mcarrano) [inline pattern-library/application-framework/multi-factor-login/design/design.md]: One thing that confuses me here is the order of the buttons. Step 1 is Next, step 2 is Login, and step 3 is Next. After the user enters their password, wouldn't the Login button change to Next is this case and then the Log In after step 3? I'm not understanding where the Next button from step 3 takes me.

File: pattern-library/application-framework/multi-factor-login/design/design.md [hunk 2]
```diff
@@ -13,17 +13,13 @@
...
+1. **More Verification Options**: Users can switch between different methods through this link if they can’t finish the current one. The link will lead to followed step.
```
Comment (mcarrano) [inline pattern-library/application-framework/multi-factor-login/design/design.md]: I think this should be marked optional as it's only needed if there are additional options.

Comment (yih-wang) [reply pattern-library/application-framework/multi-factor-login/design/design.md]: Right. Fixed it.

File: pattern-library/application-framework/multi-factor-login/design/design.md (hunk 1 above)
Comment (yih-wang) [reply pattern-library/application-framework/multi-factor-login/design/design.md]: I forgot updating the labels.. Login should be at step 3. Fixed it.

**Comment by mcarrano** [comment]
@yih-wang Do you think the screens on the Overview page should also change so the button is labeled Log In rather than Next?

**Comment by yih-wang** [comment]
@mcarrano Make sense to me. Made updates to these wireframes.

**Pull request merged**